lib/sles4sap/microsoft_sdaf_basetest.pm

```diff
@@ -11,10 +11,15 @@ use strict;
 use warnings;
 use testapi;
 use parent 'opensusebasetest';
-use sles4sap::sdaf_library;
+use sles4sap::sdaf_deployment_library;
 use sles4sap::console_redirection;
 
 sub post_fail_hook {
+    if (get_var('SDAF_NO_CLEANUP')) {
```

Comment:
Ability to keep deployment after test finished for debugging.

Reply:
It is negative logic named setting, usually I find it difficult to understand " set true something that is named _NO_ so that means false" --> maybe could be renamed something like `SDAF_RETAIN_DEPLOYMENT`

Reply:
Ok, fair point. Fixed.